cha/sota/test/setup.tex

```diff
@@ -0,0 +1,52 @@
+\subsection{Setup}
+In order to perform test these criteria, we need to have a setup to do so. 
+
+Testing the offset between two audio sources is a signal processing
+task. To make the signal processing easy and effective a clear signal is
+needed. Additionally the signal needs to be synchronized between the
+channels at recording time, since otherwise the analysis would be
+mislead by desync in the recording. Luckily most computers include
+a port for doing just that, a stereo microphone port.
+
+The stereo microphone port in the test computer is a \ac{TRS} jack
+connector, with a common ground between the two channels. To separate
+the channels into separate mono tracks we used a \ac{TRS} to \ac{RCA}
+splitter, and a custom \ac{RCA} to mono \ac{TS}, to capture the two
+separate right\jjnote{Is it right} channels from the audio sources.
+
+% Troels additions below
+Furthermore we limit our tests to a Master/Slave/Slave setup, that is;
+one phone acts as a master, and we perform the tests on two slaves
+connected to it. This makes the two devices under test as similar as
+possible, since have the reference sound. 
+Ideally we would also record the reference, the Master,
+but since we only have two input channels that is not possible.
+
+In the tests we use three Android phones:
+\begin{description}
+	\item[Master]{OnePlus One (Android 6.0.1, Kernel 3.4.112-cyanogenmod-g62d75e8)}
+	\item[Slave 1]{Moto X Style (Android 6.0.1, Kernel 3.10.84-perf-gb67345b)}
+	\item[Slave 2]{OnePlus Two (Android 6.0.1, Kernel 3.10.84-perf+)}
+\end{description}
+
+We automate the mechanical part of the testing, such as doing the
+analysis and controlling the variables. To do this we use scripts written in
+python. The scripts use the \ac{ADB} interface to control the android phones.
+Furthermore the tests are executed by a Python script to ensure consistency 
+and minimize the chances of error caused by humans.
+
+Measuring the offset between the two signals is a signal processing
+task. Within the field of signal processing the specific measure is
+called cross correlation. Cross correlation is normally
+a computationally intensive task, but due to the Convolution
+Theorem\cite{conv_theo}, it can be solved rather quickly with \ac{FFT}.
+The algorithm to calculate \ac{FFT} audio offset thus becomes doing
```

Comment:
Resten af den her paragraf er alt for specifik uden en forklaring af FFT, der bliver nød til at være en kort forklaring af hvad FFT er når du nævnter det 3 gange i en sætning.
add kilde på FFT